ui/intermittent-failures/MainView.jsx

```diff
@@ -105,6 +105,16 @@ const MainView = props => {
     } = calculateMetrics(graphData));
   }
 
+  const theadFilterThProps = (state, bug, data) => {
```

Comment:
Just a fly-by comment :) This function name is a bit confusing/vague. Could you rename it so its purpose is clearer? Something like `getHeaderAriaLabel`.

Reply:
Sure, it's done ;)